core/src/test/scala/org/broadinstitute/dsde/rawls/StatsDTestUtils.scala

```diff
@@ -59,4 +59,15 @@ trait StatsDTestUtils { this: MockitoSugar with Eventually with RawlsTestUtils =
 
   protected def expectedSubmissionStatusMetric(workspaceName: WorkspaceName, submissionStatus: SubmissionStatus, expectedTimes: Int): (String, String) =
     (s"test.workspace.${workspaceName.toString.replace('/', '.')}.submissionStatus.${submissionStatus.toString}", expectedTimes.toString)
+
+  // TODO don't repeat the code from HttpMethodExpansion.makeName() / UriExpansion.makeName()
```

Comment:
But I have to do the `uri2str()` logic somewhere.  Might as well centralize it here.

It passes because my `UriExpansion` converts `.` to `_` but now I will replace both.